client/Apache.ShenYu.AspNetCore/Utils/ShenyuCollectionExtensions.cs

```diff
@@ -0,0 +1,45 @@
+/*

```

Comment:
modify to apache license 2.0

Reply:
Resolved. Referred the license header in `incubator-shenyu` project.